packages/component-library/src/ReactMap/ReactMap.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import MapGL from 'react-map-gl';
+import './mapbox-gl.css';
+
+const ReactMap = (props) => {
```

Comment:
Might make sense to simplify the name of this component (and file) to just `Map`. Thoughts?

Reply:
That makes sense. I was trying to differentiate it from the Leaflet Map component already there. I'll rename it.